tests/jsx-significant-space/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -90,13 +101,15 @@ before_break2 = (
     <span
       barbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbarbar
     />
-    {\\" \\"}foo
+    {\\" \\"}
+    foo
   </span>
 );
 
 after_break = (
   <span>
-    foo{\\" \\"}
+    foo
+    {\\" \\"}
```

Comment:
85% of them in Facebook codebase are in the form you are enforcing in this diff, so this sounds like a good decision.